src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -57,7 +57,9 @@
     <Compile Include="System\Data\ConstraintCollection.cs" />
     <Compile Include="System\Data\ConstraintConverter.cs" />
     <Compile Include="System\Data\ConstraintEnumerator.cs" />
-    <Compile Include="System\Data\DataColumn.cs" />
+    <Compile Include="System\Data\DataColumn.cs">
```

Comment:
Nitpicking here, but are all these <subtype> tags needed?

Reply:
I think this was already in the csproj, so I think @saurabh500 might know.

Reply:
They keep popping back up, so I left them for now, it's of no use to get rid of them :(